README.md

```diff
@@ -25,7 +27,7 @@ Nothing in this spec repository is `permanent` yet. The most important pieces of
 
 Note that, as in many IPFS repositories, most of the work is happening in [the issues](https://github.com/ipfs/specs/issues/) or in [active pull requests](https://github.com/ipfs/specs/pulls/). Go take a look!
 
```

Comment:
`Install` and `Usage` sections are technically required, but that's likely a bug in the requirements, as they don't actually seem relevant here.